README.md

```diff
@@ -121,20 +128,300 @@ const source2Object = new Podium('test');
 const emitterObject = new Podium(source1Object);
 emitterObject.registerPodium(source2Object);
 
-const listener1 = function() { // normal function
+const listener1 = function(){ // normal function
+
     console.log('listener1 called');
 }
-const listener2 = function() { // another normal function
+const listener2 = function(){ // another normal function
+
     console.log('listener1 called');
 }
-emitterObject.on('test',listener1); // listener1 gets registered on emitterObject, source1Object,source2Object events
-source1Object.on('test',listener2); // listener2 gets registered on source1Object events only
+emitterObject.on('test', listener1); // listener1 gets registered on emitterObject, source1Object,source2Object events
+source1Object.on('test', listener2); // listener2 gets registered on source1Object events only
 
 source1Object.emit('test', 1); // runs all registered events
 emitterObject.emit('test', 2);
 ```
 
+#CookBook - Usage
```

Comment:
* Needs a space after the # or github won't render it as a title
* I think it is just `Cookbook` but english is not my native language as well
* I would drop the `Usage` and maybe make subtitle from it